jobbergate-api/jobbergate_api/apps/job_submissions/schemas.py

```diff
@@ -44,6 +53,7 @@ class JobSubmissionCreateRequest(BaseModel):
     job_submission_description: Optional[str]
     job_script_id: int
     slurm_job_id: Optional[int]
+    cluster_client_id: Optional[str]
```

Comment:
according to my previous comments: make it mandatory is needed

Reply:
We will make adjustments to this mechanism in a future task.

Reply:
We will make adjustments to this mechanism in a future task.